packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {DatePicker, FormGroup, ControlLabel, Form} from 'rsuite'
+import {useTranslation} from 'react-i18next'
+
+export interface DateTimeProps {
+  dateTime?: Date
+  label: string
+  changeDate(publishDate?: Date): void
+}
+
+export function DateTimePicker({dateTime, label, changeDate}: DateTimeProps) {
+  const {t} = useTranslation()
+
+  return (
+    <Form fluid={true}>
+      <FormGroup>
+        <ControlLabel>{label}</ControlLabel>
+        <DatePicker
+          style={{marginRight: 8}}
+          placement="auto"
+          value={dateTime?.getTime() === dateTime?.getTime() ? dateTime : undefined}
+          format="DD MMM YYYY"
+          ranges={[
+            {
+              label: t('timeDatePicker.today'),
+              value: new Date()
+            },
+            {
+              label: t('timeDatePicker.tomorrow'),
+              value: () => {
+                const tomorrow = new Date()
+                tomorrow.setDate(new Date().getDate() + 1)
+                return tomorrow
+              }
+            },
+            {
+              label: t('timeDatePicker.nextSaturday'),
+              value: () => {
+                const nextSaturday = new Date()
+                const i = 6 - new Date().getDay()
+                nextSaturday.setDate(i ? nextSaturday.getDate() + i : nextSaturday.getDate() + 7)
+                return nextSaturday
+              }
+            }
+          ]}
+          onChange={value => {
+            if (dateTime) {
```

Comment:
I think if you click the x inside the date/time picker you get here `value === null` and you don't want to set that. I think you can just remove the x from the pickers but maybe you still need to check here if `value` is defined.

Reply:
If I check for `value`, when it's null it disables the activity on the x button rather than clearing the form and displaying placeholder like it did before.